.github/render-action/generate.sh

```diff
@@ -40,7 +40,7 @@ function convert_markdown_to_asciidoc {
       -a lang=$language \
       -a partnums \
       -a reproducible \
-      -a revdate="$(git log -1 --pretty="format:%cs" $md)" \
+      -a revdate="$(LANG=$language git log -1 --pretty="format:%cd" --date=format:"${!DATE_FORMAT}" .)" \
```

Comment:
Technically this has nothing to do with the fonts; it is related to fixing the date of last changes displayed in the rendered files.